### PR TITLE
Use workspace limit idle timeout value in WorkspaceActivityManager

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
MultiUserWorkspaceActivityManager is now using accounts resources to retrieve workspaces idle timeouts. 

"che.limits.workspace.idle.timeout" property is also used now as default value for timeout (for single & multi user Che)
Removed old "che.workspace.agent.dev.inactive_stop_timeout_ms" property.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/8951

### CODE REVIEW

**Review by benoitf — APPROVED** [review]

**Review by mshaposhnik — APPROVED** [review]

**Comment by mkuznyetsov** [comment]
ci-test

**Review by garagatyi — APPROVED** [review]

File: multiuser/api/che-multiuser-api-workspace-activity/pom.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            <artifactId>che-core-api-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.che.core</groupId>
+            <artifactId>che-core-api-workspace</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.che.core</groupId>
+            <artifactId>che-core-api-workspace-activity</artifactId>
+            <version>6.4.0-SNAPSHOT</version>
```
Comment (skabashnyuk) [inline multiuser/api/che-multiuser-api-workspace-activity/pom.xml]: remove version

File: multiuser/api/che-multiuser-api-workspace-activity/src/main/java/org/eclipse/che/multiuser/api/workspace/activity/MultiUserWorkspaceActivityManager.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Implementation of WorkspaceActivityManager with overriden retrieval of timeout, with using
+ * Resource API to get user's limits
+ *
+ * @author Mykhailo Kuznietsov
+ */
+@Singleton
```
Comment (skabashnyuk) [inline multiuser/api/che-multiuser-api-workspace-activity/src/main/java/org/eclipse/che/multiuser/api/workspace/activity/MultiUserWorkspaceActivityManager.java:39]: test?

File: wsmaster/che-core-api-workspace-activity/src/main/java/org/eclipse/che/api/workspace/activity/WorkspaceActivityManager.java [hunk 3]
```diff
@@ -104,15 +104,20 @@ public void onEvent(WorkspaceStatusEvent event) {
...
-      } catch (ServerException e) {
-        LOG.error(e.getLocalizedMessage(), e);
       }
+    } catch (NotFoundException | ServerException e) {
+      LOG.error(e.getLocalizedMessage(), e);
     }
   }
 
+  protected long getIdleTimeout(String wsId) throws NotFoundException, ServerException {
+    return defaultTimeout;
```
Comment (skabashnyuk) [inline wsmaster/che-core-api-workspace-activity/src/main/java/org/eclipse/che/api/workspace/activity/WorkspaceActivityManager.java]: do we have a test for this class?

**Review by skabashnyuk — APPROVED** [review]

**Pull request merged**